sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -668,7 +676,7 @@ def receive_messages(
          Setting to None will fully depend on the prefetch config. The default value is 1.
         :param Optional[float] max_wait_time: Maximum time to wait in seconds for the first message to arrive.
          If no messages arrive, and no timeout is specified, this call will not return
-         until the connection is closed. If specified, an no messages arrive within the
```

Comment:
maybe we can add a NOTE/WARNING here to make it clear that setting this on receive_messages with NEXT_AVAILABLE_SESSION will not work, and they need to use the one in the constructor.

Reply:
move to docstring